$NetBSD: patch-configure,v 1.2 2019/05/31 04:45:20 gutteridge Exp $

Fix non-portable uses of test, typos in variable names, etc.

--- configure.orig	2019-04-01 10:25:27.200097504 +0000
+++ configure
@@ -14654,9 +14654,11 @@ fi
 # Check whether --enable-statusicon was given.
 if test "${enable_statusicon+set}" = set; then :
   enableval=$enable_statusicon;
+else
+  enable_statusicon=yes
 fi
 
-if (test "${enable-statusicon}" == "no"); then
+if test "x$enable_statusicon" = "xno"; then
 
 $as_echo "#define ENABLE_STATUSICON 0" >>confdefs.h
 
@@ -14666,7 +14668,7 @@ else
 $as_echo "#define ENABLE_STATUSICON 1" >>confdefs.h
 
 fi
- if test "${enable_statusicon}" != "no"; then
+ if test "x$enable_statusicon" != "xno"; then
   ENABLE_STATUSICON_TRUE=
   ENABLE_STATUSICON_FALSE='#'
 else
@@ -14684,9 +14686,11 @@ fi
 # Check whether --enable-panelapplet was given.
 if test "${enable_panelapplet+set}" = set; then :
   enableval=$enable_panelapplet;
+else
+  enable_panelapplet=yes
 fi
 
-if (test "${enable-panelapplet}" == "no"); then
+if test "x$enable_panelapplet" = "xno"; then
 
 $as_echo "#define ENABLE_PANELAPPLET 0" >>confdefs.h
 
@@ -14695,7 +14699,7 @@ else
 $as_echo "#define ENABLE_PANELAPPLET 1" >>confdefs.h
 
 fi
- if test "${enable_panelapplet}" != "no"; then
+ if test "x$enable_panelapplet" != "xno"; then
   ENABLE_PANELAPPLET_TRUE=
   ENABLE_PANELAPPLET_FALSE='#'
 else
@@ -14708,7 +14712,7 @@ fi
 # Sanity check: must build either status icon or panel applet
 #============================================================================
 
-if test "x$enable_statusicon" == "xno" && test "x$enable_panelapplet" == "xno"; then
+if test "x$enable_statusicon" = "xno" && test "x$enable_panelapplet" = "xno"; then
 	as_fn_error $? "must build either volume control status icon or volume control applet" "$LINENO" 5
 fi
 
